drivers/dma/dma_align_manager.c

```diff
@@ -0,0 +1,30 @@
+#include <nuttx/dma/dma_align_manager.h>
+#include <sys/types.h>
+int dma_align_manager_init(struct dma_align_manager_s *manager,struct dma_align_manager_init_s *initcfg)
```

Comment:
merge the alignment code into sdmmc common code, the code is too simple to become a manager.

Reply:
> merge the alignment code into sdmmc common code, the code is too simple to become a manager.

Consider the following requirement:

User layer:
write(serial port fd, user buffer, ...)
{....
   serial_port_write(buf, len)
   ....
}

Driver framework layer:
```
serial_port_write(buf, len)
{
    align_buffer = buf;
    if (buf address is not aligned)
    {
        align_buffer = alloc aligned buffer;
    }
    call chip_driver(align_buffer, len);
}
```
Therefore, anything involving DMA transfers will need to use this mechanism.